pycaret/internal/plots/time_series.py

```diff
@@ -543,16 +511,36 @@ def plot_pacf(
     data_kwargs: Optional[Dict] = None,
     fig_kwargs: Optional[Dict] = None,
 ) -> PlotReturnType:
-    """Plots the PACF on the data provided"""
-    fig, return_data_dict = None, None
-
+    """Plots the PACF on the data provided
```

Comment:
Similar changes as the `plot_acf` should be made in `plot_pacf` as well. Thanks!